snowshu/core/printable_result.py

```diff
@@ -36,8 +36,8 @@ def graph_to_result_list(graphs: nx.Graph) -> list:
             for relation in graph.nodes:
                 deps = len(nx.ancestors(graph, relation))
                 deps = " " if deps == 0 else str(deps)
-                target_sample_size = relation.population_size if\
-                    (relation.unsampled or relation.population_size < relation.sampling.size)\
+                target_sample_size = relation.population_size if \
+                    relation.unsampled or (relation.population_size < relation.sampling.size) \
```

Comment:
@eculicny I Guess this parentheses placement makes more sense?

Reply:
I also like the updated version better since it more indicative of the operands for the operators. If it makes sense to everyone else keep it in.